README_SETUP_LINUX.md

```diff
@@ -105,6 +105,12 @@ Configure `ccache` by running ([source](https://ccache.samba.org/manual.html))
 > /usr/local/bin/ccache --max-size=5G
 ```
 
+Or just:
+
+```sh
+> ccache --max-size=5G
```

Comment:
Is it not installed in /usr/local/bin in your case?

Reply:
Yes, it ended up in /usr/bin/ by default

```
09:25:56 /usr/bin  $ ./ccache 
Usage:
    ccache [options]
    ccache compiler [compiler options]
    compiler [compiler options]          (via symbolic link)
...
```

```
09:27:07 /usr/local/bin  $ ls -al
total 32
drwxr-xr-x  2 root root 4096 cze 22 16:05 .
drwxr-xr-x 12 root root 4096 cze 26 17:03 ..
-rwxr-xr-x  1 root root  216 cze 22 16:05 f2py
-rwxr-xr-x  1 root root  216 cze 22 16:05 f2py3
-rwxr-xr-x  1 root root  216 cze 22 16:05 f2py3.8
-rwxr-xr-x  1 root root  219 cze 22 16:03 pylupdate5
-rwxr-xr-x  1 root root  215 cze 22 16:03 pyrcc5
-rwxr-xr-x  1 root root  214 cze 22 16:03 pyuic5


```

Reply:
Ah, ok. We could perhaps remove the path from the instructions? Both /usr/bin and /usr/local/bin should be on the path anyway.

Reply:
Yes, I think so as well :)